README.md

```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+# space-travelers-hub
+This is  a web application for a company that provides commercial and scientific space travel services. The application allows users to book rockets and join selected space missions.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Below is the screenshot of my work
+![space-travelers-hub](https://github.com/ndikumanaisaie/space-travelers-hub/blob/use-Redux-in-React/src/assets/images/shot.png)
```

Comment:
- [ ] The link to your screenshot is broken, Kindly fix that to have a professional readme